x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/utils.ts

```diff
@@ -18,3 +18,23 @@ export const parsedPidOrEntityIdParameter = (parameters: {
     entity_id: parameters?.entityId?.[0] ?? '',
   };
 };
+
+const convertToMilliseconds = (value: number, unit: 'h' | 'm' | 's'): number => {
+  const unitToMilliseconds = {
+    h: (): number => value * 60 * 60 * 1000,
+    m: (): number => value * 60 * 1000,
+    s: (): number => value * 1000,
+  };
+
+  return unitToMilliseconds[unit]();
+};
+
+export const EXECUTE_TIMEOUT_REGEX = /^\d+(?=(h|m|s){1}$)/;
```

Comment:
Suggestion: Don't expose this RegExp (because RegExp can be stateful) and instead exposer a function `validateUnitOfTime()`?) that uses this RegExp internally to this module.